engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-traces-modal.html

```diff
@@ -0,0 +1,75 @@
+<div data-extend-template="components/views/bst-modal.html">
+  <h4 data-block="modal-header">Content Host Tracer Management</h4>
+
+  <div data-block="modal-body">
+    <form id="traceActionForm" name="traceActionForm" class="form" method="post" action="/katello/remote_execution">
+      <input type="hidden" name="remote_action" value="service_restart"/>
+      <input type="hidden" name="name" ng-value="traceActionFormValues.service"/>
+      <input type="hidden" name="host_ids" ng-value="traceActionFormValues.hostIds"/>
+      <input type="hidden" name="customize" ng-value="traceActionFormValues.customize"/>
+      <input type="hidden" name="authenticity_token" ng-value="traceActionFormValues.authenticityToken"/>
+    </form>
+
+    <div class="row">
+      <div class="col-sm-12">
+        <div bst-global-notification></div>
+      </div>
+    </div>
+
+    <div data-extend-template="layouts/partials/table.html">
+      <div data-block="search" ng-show=tracesearch></div>
```

Comment:
I downloaded and ran [`htmlhint`](https://github.com/htmlhint/HTMLHint) manually and got the following error:
```
L20 |      <div data-block="search" ng-show=tracesearch></div>
                                         ^ The value of attribute [ ng-show ] must be in double quotes. (attr-value-double-quotes)
```

After changing it to `ng-show="tracesearch"` it eliminated the error locally.  Maybe it's not processing that `</div>` because of that?

Reply:
Thanks for looking into this and finding it, everything else was in double quotes so I guess I missed this one when going through it.  Fixed